### PR TITLE
Run QA scripts which are run on the EN docs

### PR DESCRIPTION
**Pull request by Girgias**
If a change to the QA scripts are made we can at least see if this going to make the EN build fail before a push is made on the docs

### CODE REVIEW

**Comment by Girgias** [comment]
I think keeping the ``continue_on_error`` flag to true is the best as it still tries to build translations. Opinion?

**Comment by cmb69** [comment]
> I think keeping the continue_on_error flag to true is the best as it still tries to build translations. 

Ah, right, that was the intention of having it.

File: scripts/qa/extensions.xml.php [hunk 1]
```diff
@@ -154,18 +154,23 @@
     echo "{$basedir}/en/appendices/extensions.xml has been updated, check it for details\n";
 }
 
+$status = 0;
 // print the debug messages:
 if (isset($debug['membership'])) {
 	echo "\nExtensions Missing Membership:\n";
+ $status = 2;
```
[the remark this answers is not in the document]
Comment (Girgias) [reply scripts/qa/extensions.xml.php:161]: Oh this file uses tab indents >_>

**Comment by Girgias** [comment]
Closed via f0bfa37b6ae4fac314a1876fda14c89e19eeb5e5

**Pull request closed without merging**